src/services/codefixes/helpers.ts

```diff
@@ -506,6 +507,7 @@ export function createSignatureDeclarationFromCallExpression(
         contextNode,
         scriptTarget,
         NodeBuilderFlags.NoTruncation,
+        /*internalFlags*/ undefined,

```

Comment:
This makes me wonder if we are missing a test with an unresolved name for those who call this code. (Specifically fixAddMissingMember)

Reply:
I agree, that's the same thing I just responded to your last comment.